src/ajax/script.js

```diff
@@ -32,16 +32,20 @@ jQuery.ajaxPrefilter( "script", function( s ) {
 	if ( s.cache === undefined ) {
 		s.cache = false;
 	}
-	if ( s.crossDomain ) {
+
+	// These types of requests are handled via a script tag
+	// so force their methods to GET.
+	if ( s.async || s.crossDomain || s.scriptAttrs ) {
```

Comment:
Did you optimize this order for gzip compression? If not, I suspect placing `s.crossDomain` first might give us a byte or two more.

Reply:
I checked various reorderings & what you propose (+ a few others) were the smallest. Smaller by... one byte compared to my original version. 😄

+10 bytes now

Reply:
PR updated.